scripts/combine/setupCombineWMass.py

```diff
@@ -12,13 +12,13 @@
 import math
 import time
 
-scriptdir = f"{pathlib.Path(__file__).parent}"
 data_dir = common.data_dir
 
 def make_parser(parser=None):
     if not parser:
         parser = common.common_parser_combine()
```

Comment:
I guess distinguishing between common or not for the parser doesn't really make sense anymore if we only have one script. It would be good to better specify which args are relevant to the separate analyses. I think doing an initial parsing of the arguments (init_args in the common parser) could help

Reply:
Ok I have restructured the parser for setupCombineWMass